content/applications/inventory_and_mrp/purchase/manage_deals/manage.rst

```diff
@@ -2,226 +2,249 @@
-many products as you need. If the vendor sends you a confirmation or
-quotation for an order, you may record the order reference number in the
-**Vendor Reference** field. This will enable you to easily match the PO
-with the the vendor bill later (as the vendor bill will probably include
-the Vendor Reference)
+On the blank detail form, add a vendor to the :abbr:`RFQ (request for quotation)` in the
+:guilabel:`Vendor` field, and add products to the :guilabel:`Product` lines by clicking
+:guilabel:`Add a line`.
 
-.. image:: manage/manage02.png
-  :align: center
+Then, confirm the :abbr:`RFQ (request for quotation)` by clicking the :guilabel:`Confirm Order`
+button above the detail form. Doing so turns the :abbr:`RFQ (request for quotation)` into a purchase
+order.
 
-Validate the purchase order and receive the products from the Inventory
-application.
+Then, click the :guilabel:`Create Bill` button to create a vendor bill for the purchase order.
 
-Receiving Products
-------------------
+Clicking the :guilabel:`Create Bill` button reveals the :guilabel:`Draft Bill` page for the purchase
+order.
 
-If you purchased any stockable products that you manage the inventory
-of, you will need to receive the products from the Inventory application
-after you confirm a purchase order. From the **Inventory dashboard**, you
-should see a button linking you directly to the transfer of products.
-This button is outlined in red below:
+On the :guilabel:`Draft Bill`, click the :guilabel:`Edit` button to modify the bill, and add a bill
+date in the :guilabel:`Bill Date` field. If needed, add additional products to the
+:guilabel:`Product` lines by clicking :guilabel:`Add a line` in the :guilabel:`Invoice Lines` tab.
 
-.. image:: manage/manage03.png
-  :align: center
+Next, confirm the bill by clicking the :guilabel:`Confirm` button on the :guilabel:`Draft Bill`
+page.
 
-Navigating this route will take you to a list of all orders waiting to
-be received.
+.. tip::
+   Since the bill control policy is set to *ordered quantities*, the draft bill can be confirmed as
+   soon as it is created, before any products have been received.
+
+On the new :guilabel:`Vendor Bill`, add a :guilabel:`Bill Reference` number, which can be used to
+match the bill with additional documents (such as the :abbr:`PO (purchase order)`). Then, click
+:menuselection:`Confirm --> Register Payment`. Doing so causes a pop-up to appear, wherein a payment
+:guilabel:`Journal` can be chosen; a :guilabel:`Payment Method` selected; and a :guilabel:`Recipient
+Bank Account` can be selected from a drop-down menu.
 
-.. image:: manage/manage04.png
-  :align: center
+Additionally, the bill :guilabel:`Amount`, :guilabel:`Payment Date`, and :guilabel:`Memo`
+(:dfn:`Reference Number`) can be changed from this pop-up. Once ready, click :guilabel:`Create
+Payment` to finish creating the :guilabel:`Vendor Bill`. Doing so causes a green **In Payment**
```

Comment:
I think this banner could use a :guilabel: since it is an element that appears on the UI
```suggestion
Payment` to finish creating the :guilabel:`Vendor Bill`. Doing so causes a green :guilabel:`In Payment`
```